From aea843d9bb47c160ee1e793c89a3824555ba5b44 Mon Sep 17 00:00:00 2001
From: Adam Williamson <awilliam@redhat.com>
Date: Fri, 21 Feb 2014 11:46:37 -0800
Subject: [PATCH 2/2] drop baytrail setpoint to 90: temporary hack for BKO
 #70941

Since RC2 there's a pstate bug which makes performance very poor with the
default 'powersave' governor. Dirk Brandewie is working on a fix and says
dropping setpoint to 90 should work around it for now. The commit that
introduced the bug fixes another significant bug - BKO #66581 - so using
this workaround seems a better idea than reverting that commit.

This is only intended as a temporary workaround for my 'awb' kernels, it is
not for upstreaming in any way.
---
 drivers/cpufreq/intel_pstate.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/cpufreq/intel_pstate.c b/drivers/cpufreq/intel_pstate.c
index 79606f4..c8c3631 100644
--- a/drivers/cpufreq/intel_pstate.c
+++ b/drivers/cpufreq/intel_pstate.c
@@ -463,7 +463,7 @@ static struct cpu_defaults byt_params = {
 	.pid_policy = {
 		.sample_rate_ms = 10,
 		.deadband = 0,
-		.setpoint = 97,
+		.setpoint = 90,
 		.p_gain_pct = 14,
 		.d_gain_pct = 0,
 		.i_gain_pct = 4,
-- 
1.9.0

